src/casbin_actor.rs

```diff
@@ -0,0 +1,70 @@
+use actix::prelude::*;
+use casbin::prelude::*;
+use casbin::Result;
+use std::sync::Arc;
+
+pub enum CasbinCmd {
+    Enforce(Vec<String>),
+    AddPolicy(Vec<String>),
+    AddPolicies(Vec<Vec<String>>),
+    RemovePolicy(Vec<String>),
+    RemovePolicies(Vec<Vec<String>>),
+    RemoveFilteredPolicy(usize, Vec<String>),
+}
+
+impl Message for CasbinCmd {
+    type Result = Result<bool>;
+}
+
+pub struct CasbinActor {
+    enforcer: Option<Arc<async_std::sync::RwLock<Enforcer>>>,
+}
+
+impl CasbinActor {
+    pub async fn new<M: TryIntoModel, A: TryIntoAdapter>(m: M, a: A) -> Addr<CasbinActor> {
```

Comment:
A better idea here is to move `enforcer` creation into actix started and here we add two more fields for CasbinActor which is:

`model` and `actor`, but I'm not sure if the default model, adapter is clonable or not


The advantage of putting the creation code into `actix::started` is everytime when we call `ctx.stop` it'll restart the actor

Reply:
This will require `M: TryIntoModel + Clone`, `A: TryIntoAdapter + Clone`